common/culture/innovations/wc_cultural_innovations.txt

```diff
@@ -246,6 +247,23 @@ innovation_iron_star = {
 	flag = high_medieval_era_regional
 }
 
+innovation_storm_hammers = {
```

Comment:
Also, oilify images.
https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2/wiki/2D-Artist-Guidelines#innovation-icon